src/lib.rs

```diff
@@ -719,10 +729,38 @@ impl Spell {
                 section_instructions.remove(0);
                 self.process_instructions.push(Process::new(f64::from_bits(section_instructions.remove(0)) as usize, section_instructions.clone()))
             },
+            502 => {
+                self.set_meta_data(section_instructions.clone())
+            },
             _ => panic!("Invalid section")
         }
     }
 
+    fn set_meta_data(&mut self, equations: Vec<u64>) {
+        let mut codes = equations.into_iter();
```

Comment:
Equations should be renamed to attributes

```suggestion
    fn set_meta_data(&mut self, attributes: Vec<u64>) {
        let mut codes = attributes.into_iter();
```